both/helpers.js

```diff
@@ -1,322 +1,296 @@
-if(Meteor.isClient) {
-  Template.registerHelper("roundName", function(roundCode) {
+(function() { // This is a Closure
```

Comment:
Why the closure? Meteor already does this automatically. See http://staging.live.jflei.com/both/helpers.js?ba980f78122081392373894bb73156b9143d5c11.